lib/spack/docs/configuration.rst

```diff
@@ -202,12 +202,12 @@ supply ``-p`` to Spack on the command line, before any subcommands.
 .. _spack-p:
 
 ^^^^^^^^^^^^^^^^^^^
-``spack --profile``
+``spack -p``
 ^^^^^^^^^^^^^^^^^^^
```

Comment:
I do too. The only problem is that the double dash gets converted to an en dash in the [Command Index](http://spack.readthedocs.io/en/latest/command_index.html). Let me see if I can prevent those commands from even being included in the Command Index. No sense having `spack edit` and `spack edit --force`.